spec/controllers/idv/how_to_verify_controller_spec.rb

```diff
@@ -73,12 +78,28 @@
       end
 
       context 'when both ipp and opt-in ipp are enabled' do
```

Comment:
We should probably have a couple other cases, like global flag off, but SP has it enabled -- it still shouldn't show.

Reply:
Gina added this below in spec/features/idv/doc_auth/how_to_verify_spec.rb